build.sbt

```diff
@@ -84,12 +92,15 @@ lazy val publishSettings = Seq(
         <name>Jon Pretty</name>
         <url>https://github.com/propensive/magnolia/</url>
       </developer>
+      <developer>
```

Comment:
I've been using the `CONTRIBUTORS` file to manage this, though I'm not sure what is convention, or best practice, except that using a combination of both almost certainly isn't. Any ideas?

Reply:
done